internal/service/instance/nginx_instance_service_test.go

```diff
@@ -42,7 +46,7 @@ const (
 		"--with-http_v2_module --with-ipv6 --with-mail --with-mail_ssl_module --with-pcre " +
 		"--with-pcre-jit --with-stream --with-stream_realip_module --with-stream_ssl_module " +
 		"--with-stream_ssl_preread_module"
-	plusConfigArgs = "--prefix=/etc/nginx --sbin-path=/usr/sbin/nginx --modules-path=/usr/lib/nginx/modules " +
+	plusConfigArgs = "--prefix=/etc/nginx --sbin-path=/usr/sbin/nginx --modules-path=%s " +
```

Comment:
Should the `ossConfigArgs` be updated as well to make sure the change works for both plus and oss?

Reply:
We didn't have a module path in the `ossConfigArgs` and thought there might be a reason it was missing from the test so didn't change it but I can add it